db/schema_tables.cc

```diff
@@ -269,7 +269,7 @@ schema_ptr tables() {
          {"dclocal_read_repair_chance", double_type},
          {"default_time_to_live", int32_type},
          {"extensions", map_type_impl::get_instance(utf8_type, bytes_type, false)},
-         {"flags", set_type_impl::get_instance(utf8_type, false)}, // SUPER, COUNTER, DENSE, COMPOUND
+         {"flags", set_type_impl::get_instance(utf8_type, false)}, // SUPER, COUNTER, DENSE, COMPOUND, CDC
          {"gc_grace_seconds", int32_type},
          {"id", uuid_type},
```

Comment:
Better to add a boolean field to scylla_tables as this is not compatible with Cassandra.